vite.config.js

```diff
@@ -0,0 +1,12 @@
+const path = require('path');
+
+module.exports = {
+  root: 'demo',
+  outDir: 'demo_dist',
+  alias: {
+    '/@src/': path.resolve(__dirname, 'src'),
```

Comment:
Rather something more standard like

```suggestion
    '@/': path.resolve(__dirname, 'src'),
```